source/fundamentals/connection/connect.txt

```diff
@@ -95,6 +95,14 @@ verify that the connection is successful:
 .. literalinclude:: /code-snippets/connection/stable-api-connect.js
    :language: javascript
 
+.. note::
+
+   As of v4.7, the {+driver-short+} will automatically call the ``MongoClient.connect()``
+   method when using the client to interact with your MongoDB instance, **except** when
```

Comment:
I: there was a whole thing about a year ago about server vs. deployment vs. instance. i think the upshot was:

- MongoDB Server = the product
- MongoDB deployment = a running instance of MongoDB
- MongoDB instance = not preferred

```suggestion
   method when using the client to interact with your MongoDB deployment, **except** when
```